settings.py

```diff
@@ -2089,5 +2092,3 @@ def _pkce_required(client_id):
 GOOGLE_SHEETS_API_VERSION = "v4"
 
 DAYS_KEEP_GSHEET_STATUS = 14
```

Comment:
There are many settings down here that cannot be overridden by localsettings. I wonder if the authors knew that? I think the only thing that should be down here are calculated values based on other settings, although it could be argued that even those things should not live in this file. Looks ripe for cleanup.

Reply:
Yeah I noticed those others but didn't have a plan for how to approach them yet. I did a brief pass to see obvious settings that should be set before importing, and the only problematic one appeared to be `ALLOW_MAKE_SUPERUSER_COMMAND` since it is set to False on SaaS envs ([example](https://github.com/dimagi/commcare-cloud/blob/master/environments/production/public.yml#L139)) but when querying from a shell on that env, it is True.